packages/genesys-spark-components/src/components/stable/gux-list/gux-list-item/gux-list-item.tsx

```diff
@@ -18,27 +16,6 @@ export class GuxListItem {
   @Prop()
   disabled: boolean = false;
 
-  @Listen('mouseup')
```

Comment:
@daragh-king-genesys I am tagging you here because you seem to be the one who added this code. Can you remember if this code has any utility that I may be overlooking? I am removing it because it is causing unwanted focus behavior after my changes in the `gux-list` file.

Reply:
Sorry I missed this comment at the time Katie. I assume this is why you have not merged.
I have no idea and happy to remove it if you think it is causing issues.

While I was looking into it I noticed the same code is in a few other list item like components:
- gux-month-list-item
- gux-rich-highlight-list-item
- gux-rich-style-list-item

We will need an issue to look into removing the code from there too as I assume they are based on this original component.